code/modules/mob/living/living.dm

```diff
@@ -564,6 +564,7 @@
 		else
 			if(!silent)
 				to_chat(src, span_notice("You lay down."))
+				playsound(loc, 'goon/sounds/body_thud.ogg', ishuman(src) ? 40 : 15, 1, 0.3)
```

Comment:
Silent seems to be related to being intentional, not to silencing feedback. This should probably be moved outside of this check.